packages/paymasters/src/PermitPaymasterUtils.ts

```diff
@@ -30,15 +32,25 @@ export const UNISWAP_V3_QUOTER_CONTRACT_ADDRESS = '0xb27308f9F90D607463bb33eA1Be
 export const SWAP_ROUTER_CONTRACT_ADDRESS = '0xE592427A0AEce92De3Edee1F18E0157C05861564'
 export const GSN_FORWARDER_CONTRACT_ADDRESS = '0xAa3E82b4c4093b4bA13Cb5714382C99ADBf750cA'
 export const UNISWAP_V3_DAI_WETH_POOL_CONTRACT_ADDRESS = '0xC2e9F25Be6257c210d7Adf0D4Cd6E3E881ba25f8'
+export const UNISWAP_V3_DAI_WETH_2_POOL_CONTRACT_ADDRESS = '0x60594a405d53811d3BC4766596EFD80fd545A270'
```

Comment:
it is very bad to hard-code contract addresses (and so many of them) into the code. this works for exactly one network..

Reply:
These are used for tests only

Reply:
In this case I believe they should be defined in the tests or package themselves. Or, we may need to check in a `networks.json` file with supported contracts across networks.